File: AccountService/src/main/java/com/example/accountservice/request/RegisterRequest.java
package com.example.accountservice.request;

import lombok.*;

import java.math.BigDecimal;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class RegisterRequest {
    private String code;

    private String email;

    private String personalEmail;

    private String fullname;

    private String password;

    private String phone;

    private BigDecimal salary;

    private String image;

    private Long roleId;

    private Long departmentId;
}

File: AuthService/Requests/RegisterRequest.cs
using System.Runtime.CompilerServices;
using System.Runtime.InteropServices;

namespace AuthService.Requests
{
    public class RegisterRequest
    {
        public string Code { get; set; }

        public string Email { get; set; }

        public string PersonalEmail { get; set; }

        public string Fullname { get; set; }

        public string Password { get; set; }

        public string Phone { get; set; }

        public decimal Salary { get; set; }

        public string? Image { get; set; } = "";

        public long RoleId { get; set; }

        public long DepartmentId { get; set; }
    }
}
